webapi/src/main/scala/org/knora/webapi/messages/StringFormatter.scala

```diff
@@ -577,7 +578,7 @@ class StringFormatter private (
   initForTest: Boolean = false
 ) {
 
-  import StringFormatter._
+  import org.knora.webapi.messages.StringFormatter._
```

Comment:
Could that be moved to the beginning of the file as Codacy reports?

Reply:
done during the merge with main.